### PR TITLE
Add example: uploading files in parallel to GCS

### PR DESCRIPTION
**Pull request by fredo838**
This PR adds an example, one that uploads files to GCS in parallel

### CODE REVIEW

**Comment by fredo838** [comment]
@omarryhan

**Comment by omarryhan** [comment]
Thank you Fredo, much appreciated!

I think the example should work without replacing the URL of the request(s). Aiogoogle first checks if there's an `upload _path` passed. If it finds one, it automatically uses the upload URL instead of the ordinary URL. Did you try without it?

File: examples/upload_storage_files_parallel.py [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    async with aiogoogle_factory:
+        storage = await aiogoogle_factory.discover("storage", "v1")
+        requests = []
+        for file_name, file_path, mime_type in file_list:
+            req = storage.objects.insert(bucket=bucket, name=file_name, upload_file=file_path)
+            # add the mime_type to the headers. You can autodetect mimetypes using python's
+            # built in `mimetypes` library
+            req.headers = {'Content-Type': mime_type}
+            # the upload endpoint for GCS is special, hence the replace
+            req.uri = req.url.replace('storage.googleapis.com/storage',
```
Comment (omarryhan) [inline examples/upload_storage_files_parallel.py:42]: This is the line I'm referring to

Comment (fredo838) [reply examples/upload_storage_files_parallel.py]: I removed it (I amended the commit by accident so it's still a single commit).

**Pull request merged**